spec/unit/webrtc/groupCall.spec.ts

```diff
@@ -136,11 +136,17 @@ class MockCall {
     public state = CallState.Ringing;
     public opponentUserId = FAKE_USER_ID_1;
     public callId = "1";
+    public localUsermediaFeed = {
+        setAudioVideoMuted: jest.fn<void, [boolean, boolean]>(),
+        stream: new MockMediaStream("stream"),
```

Comment:
No tracks?

Reply:
Actually, wouldn't this break the tests?

Reply:
We don't need them currently, so seemed no point in adding them.